homeassistant/components/doods/image_processing.py

```diff
@@ -231,7 +230,9 @@ def device_state_attributes(self):
         }
 
     def _save_image(self, image, matches, paths):
-        img = Image.open(io.BytesIO(bytearray(image))).convert("RGB")
+        img = convert_to_pil_image(image)
```

Comment:
We could just pass the PIL image to _save_image actually

Reply:
Actually this doesn't matter as we have already validated the image data in `process_image`

Reply:
I think pil_image would be a better variable name..?